### PR TITLE
XOT-968 - Add security fixes

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -143,6 +143,8 @@
 # Django's default value for LANGUAGE_COOKIE_DOMAIN is None
 LANGUAGE_COOKIE_DOMAIN = env.str('LANGUAGE_COOKIE_DOMAIN', None)
 
+CSRF_COOKIE_HTTPONLY = True
```
Comment (sdonk) [inline conf/settings.py]: This is already enabled https://github.com/uktrade/great-domestic-ui/blob/develop/conf/settings.py#L325

File: finance/views.py [hunk 2]
```diff
@@ -22,6 +23,7 @@ class TradeFinanceView(
     slug = slugs.GREAT_GET_FINANCE
 
 
+@method_decorator(never_cache, name='dispatch')
```
Comment (sdonk) [inline finance/views.py]: What's the reason not to cache this view?

Comment (eadpearce) [inline finance/views.py]: why is this needed?

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,7 @@
 - XOT-977 - Add marketing campaigns form, original clone of community join-form
 - No ticket - Consistent use of breadcrumbs component
 - TT-1678 - Rename EU exit to Brexit
+- XOT-968 - Fix security issues raised by latest pen test
```
Comment (sdonk) [inline CHANGELOG.md]: Because of the recent changelog update this is not in the incorrect place

File: conf/settings.py (hunk 1 above)
Comment (WillTaylor22) [reply conf/settings.py]: HTTPONLY and SECURE are different... are you sure this is redundant?

Comment (richtier) [reply conf/settings.py]: confirmed not redundant :) https://docs.djangoproject.com/en/2.2/ref/settings/#csrf-cookie-httponly

Comment (WillTaylor22) [reply conf/settings.py]: Done thanks
